src/diagnosticMessages.json

```diff
@@ -77,6 +77,7 @@
   "A parameter property cannot be declared using a rest parameter.": 1317,
 
   "Duplicate identifier '{0}'.": 2300,
+  "Identifier '{0}' is re-exported from modules '{1}' and '{2}'.": 2301,
```

Comment:
The convention for new errors that do not exist in TS I used so far is to use a 3-digit code instead (TS has 4 digit codes only), in this case that'd be the next `2XX` one.

For reference, the original TS error here is

```json
    "Initializer of instance member variable '{0}' cannot reference identifier '{1}' declared in the constructor.": {
        "category": "Error",
        "code": 2301
},
```

Reply:
Looks like TS already has a diagnostics for this situation, so I'll use that instead of creating an AS-specific one.